lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
```

Comment:
Are we going to make this part of the official coding style?
i.e. Ban syntax like:
```js
function errorMsg(code) {
  return code + '_ERROR';
}
```

Also, how do people feel about the backtick + dollar syntax? Reminds me of PHP. I think I prefer explicit concatenation but "meh".
I suppose I just think we should be careful about using features just because they are new in ES6. I'm sure not all of them will end up in the next edition of "The good parts"! ;-)